AsyncDisplayKit/ASNetworkImageNode.mm

```diff
@@ -193,6 +210,23 @@ - (UIImage *)defaultImage
   return _defaultImage;
 }
 
+- (void)setAnimatedImage:(id<ASAnimatedImageProtocol>)animatedImage
+{
+  if (_needsToApplyPendingContentMode) {
```

Comment:
As @Adlai-Holler said, these two methods need to be guarded with the instance lock.